packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -1010,14 +1010,13 @@ describe('plugin-meetings', () => {
             state: 'JOINED',
           };
           meeting.areVoiceaEventsSetup = false;
-          meeting.roles = ['COHOST'];
```

Comment:
Should we keep this so that it's clear that it's not a host and even then caption can be enabled?

Reply:
I don't think that's required because we do not have any conditions at all and we check that the turnOnCaptions is called